build

```diff
@@ -3,20 +3,14 @@ set -euo pipefail
 
 TARGET="${1:-default}"
 
+scripts/info-branch-commit
+export readonly TARGET=$(scripts/option-prompt "Select a target" vm develop validation master lab-key shed-key stuck default)
```

Comment:
I am not necessarily against an interactive script, but I wouldn't want that to be the only option, because

- I want to be able to script things,
- I want to be able to run a combined command like `./build vm; result/bin/run-in-vm` from history without having to type extra stuff into the terminal.

It would be fine for `./build` to prompt a target if none is given, I think?